llvm/test/CodeGen/X86/2006-10-02-BoolRetCrash.ll

```diff
@@ -2,7 +2,7 @@
 ; PR933
```

Comment:
There shouldn't be tests in test/CodeGen/X86 using the default target triple... probably the tests should be relocated or fixed.

Reply:
Handling of these tests here: https://github.com/llvm/llvm-project/pull/101944.

Reply:
(committed now)